site/docs/master/code-standards.md

```diff
@@ -0,0 +1,61 @@
+# Code Standards
+
+## Adding a changelog
```

Comment:
This content (adding a changelog, as well as DCO signoff) also exists in `CONTRIBUTING.md` - I'd say if we're moving coding standards into a separate doc on the website, we probably want to replace the existing content in `CONTRIBUTING.md` with just a pointer to this section/page in the docs - WDYT?

Reply:
Yes, good point!